pyproject.toml

```diff
@@ -1,5 +1,123 @@
 [build-system]
-requires = ["setuptools >= 38.0.0",
-            "setuptools_scm >= 2.0.0",
+requires = ["setuptools >= 61.2",
+            "setuptools_scm[toml]>=7.1",
             "wheel"]
```

Comment:
BTW, "wheel" is not needed in build-system.requires, modern setuptools adds it itself when it is needed (and, obviously, it is needed only when wheels are built).

Reply:
good catch!